### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by JsBergbau** [hinchliff/rpi2-mongodb-compile#1]
Hi hincchliff,

just wanted to say thank you for your detailed instruction of how to compile mongodb 3.12.12

**Comment by hinchliff** [comment]
You're welcome!  I'm glad it was useful to you -- looks like you're using some of the instructions over at [JsBergbau/Omada-SDN-Controller-mongodb-Raspberry-PI](https://github.com/JsBergbau/Omada-SDN-Controller-mongodb-Raspberry-PI)?  Awesome!

**Comment by JsBergbau** [comment]
Yes I used your instructions to compile mongodb and also made this build available for download. Also of course I've linked to your repository.
So users with less time and/or experience can directly download mongodb from my repository https://github.com/JsBergbau/Omada-SDN-Controller-mongodb-Raspberry-PI
Your instructions were very useful. Without your help, probably I hadn't managed to run latest controller on RaspberryPI. I know that there is a docker container. However I'm running the controller within firejail to run it from a different IP. Using that together with docker is probably hard to achieve, perhaps even not possible.

**Comment by hinchliff** [comment]
Right, I don't run the docker image from this repo.  I just used it to compile MongoDB, and I copy the binary from this repo into my other [rpi2-unifi-controller](https://github.com/hinchliff/rpi2-unifi-controller) image that I run.

I did go ahead and add more instructions on using the docker image from this repo to extract the binaries.  And I added the binary as a [Release](https://github.com/hinchliff/rpi2-mongodb-compile/releases) artifact. 

But I think I might encourage other people to compile on their own, instead of blindly downloading binaries from strangers on the internet :)
